magento/magento2
# #29089 : GraphQL: category & categoryList query return wrong image url

### Description (*)

This PR will fix the issue with the category image URL if the url rewrite is disabled in magento
### Related Pull Requests


### Fixed Issues (if relevant)

Fixes https://github.com/magento/magento2/issues/29089

### Manual testing scenarios (*)

Make sure the url rewrite is disabled in Magento 

Run following query
```
 {
  category(id: 24) {
    image
  }
}
```
### Expected Result
![image](https://user-images.githubusercontent.com/39480008/97017522-a6db3580-156b-11eb-9682-52d5ad3ba2f5.png)

### Questions or comments


### Contribution checklist (*)
